contracts/periodic-prize-pool/PeriodicPrizePool.sol

```diff
@@ -386,39 +391,53 @@ abstract contract PeriodicPrizePool is Timelock, BaseRelayRecipient, ReentrancyG
     return FixedPoint.calculateMantissa(_balanceOfTicketInterest(user, tickets), tickets);
   }
 
-  function redeemTicketsInstantly(uint256 tickets, bytes calldata data) external nonReentrant returns (uint256) {
+  function redeemTicketsInstantly(
+    uint256 tickets, 
+    bytes calldata data,
+    bytes calldata operatorData
```

Comment:
This isn't an operator function, so operatorData isn't necessary here.

Reply:
done